docs/src/js/main.js

```diff
@@ -193,6 +193,55 @@ document.querySelectorAll('.dialog-button').forEach(function (el) {
     dialogWidget._el.addEventListener('dialog-cta', logEvent);
 });
 
+// EXPRESSIVE-LOADER
+// Simple expressive loader implementation that handles rotating messages.
+class ExpressiveLoader {
```

Comment:
@ArtBlue Here's a simple JS implementation of the message rotation. There's more nuance I'd bring to a robust implementation (in ebayui-core and/or makeup-js) around timing, reduced motion enhancements, and start states, but opted to just do message rotation for the Skin examples.